packages/edit-post/src/components/sidebar/settings-header/index.js

```diff
@@ -1,65 +1,73 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
 /**
  * WordPress dependencies
  */
 import { Button } from '@wordpress/components';
-import { __ } from '@wordpress/i18n';
-import { withDispatch } from '@wordpress/data';
+import { __, sprintf } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+// translators: Label for the Block Settings Sidebar tab, not selected.
+const BLOCK_LABEL = __( 'Block' );
+// translators: Label for the Block Settings Sidebar tab, selected.
+const BLOCK_SELECTED_LABEL = __( 'Block (selected)' );
```

Comment:
Why not keep these inline?

Reply:
The strings were appearing at multiple places. Hence, I made it into a constant to re-use across the file. Following Miguel's suggestion.

Reply:
Ref to my comment: https://github.com/WordPress/gutenberg/pull/17311#discussion_r447819934

My main concern was the string interpolation, that was the one thing that needed to be fixed. As for reusing the string or inlining it, I'd say that's up to you (my personal preference is probably inlining).

Reply:
I can put it back inline.

Side question out of curiosity: If I add same string again in the same file (or maybe anywhere else), do I have to add the `translators: ...` comment again?

Reply:
b3a060e

Reply:
Honestly I am not sure about the translators comment :)